docker-compose.yaml

```diff
@@ -173,31 +165,27 @@ services:
     depends_on:
       kratos-postgres:
         condition: service_started
-    restart: "no"
-
+    restart: 'no'
   kratos-postgres:
     image: docker.io/library/postgres:13-alpine
     environment:
       - POSTGRES_PASSWORD=${KRATOS_POSTGRES_PASSWORD}
       - POSTGRES_USER=kratos
       - POSTGRES_DB=kratos
     restart: unless-stopped
-    volumes:
-      - kratos-db:/var/lib/postgresql/data
+    volumes: [kratos-db:/var/lib/postgresql/data]
     healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      test: [CMD-SHELL, pg_isready -U kratos]
       interval: 5s
       timeout: 5s
       retries: 5
-
-  # This is insecure, do not use this in production or on the public internet
-  # mailslurper:
-  #   image: docker.io/oryd/mailslurper:smtps-latest
-  #   ports: # port 1025 is used internally for smtp, you do not have to expose that
-  #     - 4436:4436 # web UI
-  #     - 4437:4437 # http API
-  #   restart: unless-stopped
-
+      # This is insecure, do not use this in production or on the public internet
```

Comment:
this indent is faulty

Reply:
Was this formatted using prettier? I believe the root problem is that you haven't set up prettier properly.

Reply:
will fix in later PR